hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupUtil.java

```diff
@@ -34,24 +34,29 @@
   private RSGroupUtil() {
   }
 
-  @FunctionalInterface
-  private interface GetRSGroup {
-    RSGroupInfo get(String groupName) throws IOException;
-  }
-
-  private static Optional<RSGroupInfo> getRSGroupInfo(MasterServices master, GetRSGroup getter,
+  /**
+   * Will try to get the rsgroup from {@link TableDescriptor} first, and then try to get the rsgroup
+   * from the {@link NamespaceDescriptor}. If still not present, return empty.
```

Comment:
Seems the impl didn't get rsgroup from NamespaceDescriptor?

Reply:
See below, you need to show all the code, include the unmodified pieces.